tweb/src/main/resources/templates/index.html

```diff
@@ -83,7 +84,37 @@ <h1 class="home-title">
                 </div>
             </div>
         </div>
+        <div class="container my-5 px-5">
+            <div class="row">
+                <div class="col-12 col-md-3 col-lg-3 offset-md-2">
+                    <img style="width: 87%;"
+                         src="https://user-images.githubusercontent.com/6882032/41536582-52e43a30-7306-11e8-92df-2fe6502ad760.png"/>
```

Comment:
The image should be put in `tweb/src/main/frontend/src/img/` and then be referenced like `th:src="@{/assets/img/<image-name>.png}"` in the tag

Reply:
Also the white background of the image should be transparant, we'll supply you with a new image to use

Reply:
@rsercano we've updated the image in the issue, so you can use that

Reply:
done